WordPress/Sniff.php

```diff
@@ -175,65 +175,65 @@ abstract class Sniff implements PHPCS_Sniff {
 	 * @var array
 	 */
 	protected $autoEscapedFunctions = array(
-		'allowed_tags'              => true,
```

Comment:
Hmm... this change shouldn't be here.... (doesn't belong in this PR. You've probably run phpcbf, but this was fine as it was.)